src/tools/tidy/src/ui_tests.rs

```diff
@@ -186,29 +196,19 @@ pub fn check(root_path: &Path, bless: bool, bad: &mut bool) {
     // if there are any file names remaining, they were moved on the fs.
     // our data must remain up to date, so it must be removed from issues.txt
     // do this automatically on bless, otherwise issue a tidy error
-    if bless && !remaining_issue_names.is_empty() {
-        let issues_txt_header = r#"
-/*
-============================================================
-    ⚠️⚠️⚠️NOTHING SHOULD EVER BE ADDED TO THIS LIST⚠️⚠️⚠️
-============================================================
-*/
-[
-"#;
```

Comment:
I genuinely do not think we should remove this.

Reply:
I understand the concern but I'm not sure how this will help. If someone adds a line to the end of this file, reviewers won't be able to see this header anyway.

How about hardcoding the current length and checking if it's not equal to that? This way we will continuously be aware of it without relying on reviewers' mistakes.

Reply:
It is not for informing a reviewer.

`const ISSUES_ENTRY_LIMIT` is already checked.

Reply:
> It is not for informing a reviewer.

It has to. When someone accidentally adds a line, reviewer should be aware of this information. 

> const ISSUES_ENTRY_LIMIT is already checked.

What I mean is this:

```diff
diff --git a/src/tools/tidy/src/ui_tests.rs b/src/tools/tidy/src/ui_tests.rs
index 5b3b948d4a1..86e72823036 100644
--- a/src/tools/tidy/src/ui_tests.rs
+++ b/src/tools/tidy/src/ui_tests.rs
@@ -100,6 +100,9 @@ fn check_entries(tests_path: &Path, bad: &mut bool) {
 }

 pub fn check(root_path: &Path, bless: bool, bad: &mut bool) {
+    // This can be decreased but can never be increased.
+    const EXPECTED_LINE_COUNT: usize = 4372;
+
     let path = &root_path.join("tests");
     check_entries(&path, bad);

@@ -120,6 +123,8 @@ pub fn check(root_path: &Path, bless: bool, bad: &mut bool) {
         })
         .collect();

+    assert_eq!(EXPECTED_LINE_COUNT, allowed_issue_names.len(), "If you have added new lines to issues.txt, please revert them; otherwise, decrease the EXPECTED_LINE_COUNT.");
+
     if !is_sorted && !bless {
         tidy_error!(
             bad,
```

Reply:
> It has to. When someone accidentally adds a line, reviewer should be aware of this information.

The vast majority of the information in the Rust repository is addressed directly at the contributor, and only incidentally useful to a reviewer, so I disagree with this "has to" assertion.

Reply:
Anyways, the reason there's nothing exactingly doing `assert_eq!` is because there was a request in the original PR to make this list `--bless`able, to reduce annoyance. So I implemented that. Making sure this number `--bless`es only up and not down would require a lot of fiddly logic, I would think? And it would also invite merge conflicts on the value a lot.